diary/urls.py

```diff
@@ -9,4 +9,6 @@
     path('/<int:diaryId>/graph', GetNodeData.as_view()),
     path('/user/<int:userId>', GetDiaryByUserView.as_view()),
     path('/checkanswer', CheckAnswerView.as_view()),
+    path('/checkdiary', IsExistDiaryView.as_view()),
```

Comment:
checkdiary 에서 diary는 중복이라 빼도 될듯함

```suggestion
    path('/check', IsExistDiaryView.as_view()),
```